apps/bmd/src/utils/tallies.ts

```diff
@@ -6,10 +8,114 @@ import {
   VotesDict,
   YesNoVote,
 } from '@votingworks/types'
-import { CandidateVoteTally, Tally, YesNoVoteTally } from '../config/types'
+import {
+  CandidateVoteTally,
+  MsEitherNeitherTally,
+  Tally,
+  YesNoVoteTally,
+} from '../config/types'
 import { computeTallyForEitherNeitherContests } from './eitherNeither'
 import { getSingleYesNoVote } from './votes'
 
+const combineCandidateTallies = (
+  tally1: CandidateVoteTally,
+  tally2: CandidateVoteTally
+): CandidateVoteTally => {
+  const candidates = []
```

Comment:
Give it an annotation so it isn't inferred as `any[]`:
```suggestion
  const candidates: number[] = []
```